.github/workflows/nessie.yaml

```diff
@@ -45,3 +45,19 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.NESSIE_AWS_SECRET_ACCESS_KEY }}
         run: |
             docker-compose ps -q postgres && docker-compose exec postgres pg_dumpall --username=lakefs | gzip | aws s3 cp - s3://nessie-system-testing/${{ github.run_number }}/dump.gz
+      - name: Run lakeFS GS
```

Comment:
Don't you prefer to run this in parallel? There's no need to wait for S3 to finish.
Also, It won't run if S3 failed. Might have some added value to know if both failed, or just S3.

Reply:
can't run steps in parallel - will open task for passing docker image build as it the step that takes the longest.
running the GS test takes less than half a min so I'm fine with that for now.